src/ssh_keydir/ssh_keydir_test.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ssh_keydir
+
+import (
+	"testing"
+)
+
+func Test(t *testing.T) {
+	// TODO
```

Comment:
As a starting point you can translate these from python: https://github.com/coreos/init/blob/master/tests/test_update_ssh_keys.py